main.js

```diff
@@ -0,0 +1,179 @@
+// Exerciess 1: in một bảng của một số bất kì với các số từ 1 đến 10 và hiển thị ra kết quả 
+const multiplicationTable = num =>{
+    if (num < 1 || num > 10) return ;
+
+    console.log('Bảng tính nhân của:', num, 'là');
+    for (let i = 1; i <= 10; i++) {
+        return console.log( num, 'x', i,'=', i * num);
+    }
+}
+
+/**
+ * Exerciess 2:
+ * Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 
+ * In các số chẵn từ 1 đến n Vd n = 6: in ra: 2 4 6
+ */
+const findEvenNumber = (num) => {
+    if ( 1 <= num <= 30) {
+        for (let i = 1; i <= num; i++) { 
+            if (i % 2 === 0)
+            console.log(i);
+        }
+    }    
+}
+
+/**
+ * Exerciess 3: 
+ * Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 Tính tổng từ 1 đến n
+ * vd nhập vào 6: Tổng = 1 + 2 + 3 + 4 + 5 + 6 = 21
+ */
+
+const sum = num => {
+    if (num < 1 || num > 30) return 'Hãy điền 1 số từ 1 đến 30'
+
+    let sum = 0
+    if ( 1 <= num <= 30 ) {
+        for (let i = 1; i <= num; i++) {
+            sum += i
+        }
+    }  
+    return console.log('Tổng từ 1 đến', num, 'là', sum)
+}
+
+/**
+ * Exerciess 4: 
+ * Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 
+ * In ra n! (giai thừa của n)
+ */
+
+const calculateFactorial = num => {
+    let factorial = 1
+
+    if ( 1 <= num <= 30 ) {
+        for (let i = 1; i <= num; i++) {
+            factorial *= i
+        }
+    }  
+    return console.log('Giai thừa của', num, 'là', factorial)  
+}
+
+/**
+ * Exerciess 5: 
+ * Hàm nhận vào 1 mảng đếm xem trong mảng có bao nhiêu số chẵn
+ * Vd: param là [1, 2, 3, 6, 2, 9, 0] thì return 4 vì có 4 số chẵn
+ */
+
+const getEvenNum = () => {
+    let evenNum = 0
+    array.forEach(num => {
+      if (num % 2 === 0) {
+        evenNum++
+        }
+    })
+    return console.log('có:', evenNum, 'số chẵn');
+}
+
+console.log('Check Ex_5');
+array = [1,4,56,78,43,23,45]
```

Comment:
```suggestion
const array = [1, 4, 56, 78, 43, 23, 45]
```
Khi khai báo bắt buộc phải dùng từ khóa
Nhìn kỹ cách đặt dấu cách